options/posix/generic/posix_time.cpp

```diff
@@ -30,3 +36,8 @@ int futimes(int, const struct timeval[2]) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
+int lutimes(const char *, const struct timeval[2]) {
```

Comment:
`lutimes` is the same as `utimes` but with `AT_SYMLINK_NOFOLLOW` passed as the flags for `sys_utimensat`